streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaStreamingContext.scala

```diff
@@ -578,6 +578,14 @@ class JavaStreamingContext(val ssc: StreamingContext) extends Closeable {
     ssc.addStreamingListener(streamingListener)
   }
 
+
+  /**
+   * Return the current state of the context.
+   */
+  def getState(): StreamingContextState = {
```

Comment:
Not sure Java can handle this. Also, please add a test to `JavaStreamingContextSuite`